### ISSUE TITLE
Difference between Packery and Isotope

### ISSUE THREAD

**Issue by guyisra** [metafizzy/packery#5]
Since isotope was made a long time ago, and many have purchased the license for it (as do I) I wonder - what is the difference between packery and isotope? 

from packery point of view it is draggable and fills in the gaps, which isotope has trouble doing. 

Should isotope users abandon it and move to packery? Will isotope still be maintained, or will it ever remain in the shadows of its better looking, cooler draggable successor (is packery isotope's successor? or are they destined to rule the bin-packingland side by side?) 

Basically, my worry is that isotope will fade from existence since packery is shinier..

I'd love it if you can shed some light on it. 

Thanks!
